www/boot.js

```diff
@@ -31,7 +31,7 @@ function resetLocalEndpoint() {
     function getDefaultAppId() {
         var defaultAppID;
         if(window.cordova) {
-          defaultAppID = 1006;
+          defaultAppID = 11780;
```

Comment:
To differentiate the users that upgrade to the new endpoint (frontend.binary) in the android app, @ashkanx has created a new app id.